apps/web-giddh/src/app/theme/add-bulk-items/add-bulk-items.component.ts

```
@@ -232,6 +232,54 @@ export class AddBulkItemsComponent implements OnInit, OnDestroy {
         this.updateFocusStates(focusableElements);
     }
 
+    /**
+     * Handles global escape key press to close the dialog
+     *
+     * @memberof AddBulkItemsComponent
+     */
+    public handleGlobalEscape(): void {
+        this.dialogRef?.close();
+    }
```

Biome (2.1.2)

[error] Duplicate class member name "handleGlobalEscape"



(lint/suspicious/noDuplicateClassMembers)